### ISSUE TITLE
Star-imports don't get included in the bundle

### ISSUE THREAD

**Issue by amsyarasyiq** [timocov/dts-bundle-generator#304]
**Bug report**
When star-importing another module `import *`, the value gets ignored and won't bundle. The following will error out without `--no-check`:

**Input code**

```ts
// otherLib.ts
export function foo_func() {}
export function bar_func() {}

// index.ts
import * as lib from "./otherLib";
export const Lib = lib;
```

**Expected output(?)**

```ts
export declare const Lib: {
	foo_func(): void;
	bar_func(): void;
};

export {};
```

**Actual output**

```ts
// without --no-check: dist/index.d.ts(1,34): error TS2552: Cannot find name 'lib'. Did you mean 'Lib'?
export declare const Lib: typeof lib;

export {};
```

**Additional context**
This can be work around by "cloning" the imported module with spread operator `{ ...lib }`. Doing this will result the "expected output" shown above.

**Comment by timocov** [comment]
As a workaround for now you can use something like `export { lib as Lib }`, technically should be around the same. I'll take a look in details later

**Comment by amsyarasyiq** [comment]
> What version of the tool you're using?

From package.json, `"dts-bundle-generator": "^9.3.1",`

My usage was something along with:
```ts
export const getSomething = () => ({
	somethingKey: starImportedModule
})
```

**Comment by timocov** [comment]
The fix has been [published in 9.4.0 version](https://github.com/timocov/dts-bundle-generator/releases/tag/v9.4.0).
